lib/interpolation.ts

```diff
@@ -11,32 +18,30 @@ const delimiters = {
   },
 };
 
-const isHelperIdentifier = identifier => {
+function isHelperIdentifier(identifier: string): boolean {
   return (
     identifier.startsWith(delimiters.helpers.start) ||
     identifier.startsWith(delimiters.helpers.end)
   );
-};
+}
 
-const generateReplaceFn = (matchedText, startIndex, replacementString) => {
+function generateReplaceFn(
+  matchedText: string,
+  startIndex: number,
+  replacementString: string
+): (arg0: string) => string {
```

Comment:
Curious about this `arg0` pattern you've been using for returned functions. In LDL we just used the variable name in the type (so `currentStringValue` here). Not opposed to doing it this way, just wondering if there was a reason you made the switch

Reply:
It was GitHub Copilot's suggestion 😬 Per our Huddle on Friday, I'm changing `arg0` to something more descriptive.